desktop/src/com/mygdx/game/client/di/SetterBindingsModule.java

```diff
@@ -0,0 +1,34 @@
+package com.mygdx.game.client.di;
+
+import com.mygdx.game.client.ecs.entityfactory.ChooseableSetter;
+import com.mygdx.game.client.ecs.entityfactory.DesktopCoordinateSetter;
+import com.mygdx.game.client.ecs.entityfactory.ModelInstanceCompSetter;
+import com.mygdx.game.client.ecs.entityfactory.NavigationDirectionSetter;
+import com.mygdx.game.client.ecs.entityfactory.TextureCompSetter;
+import com.mygdx.game.client_core.ecs.entityfactory.Setter;
+import dagger.Binds;
+import dagger.Module;
+import dagger.multibindings.IntoSet;
+
+@Module
+public interface SetterBindingsModule {
+  @Binds
+  @IntoSet
+  Setter providesChooseableSetter(ChooseableSetter setter);
```

Comment:
@IntoSet sprawia, że taki Setter będzie dostarczany do kolekcji Set<Setter>, wtedy w kodzie Handlera mogę już nie myśleć o dopisywaniu kolejnych Setterów jako parametry kontruktora, bo Dagger sam je agreguje

tylko trzeba dodawać je tutaj do Module, więc to jednak coś za coś